smart_control/simulator/building_radiation_utils_test.py

```diff
@@ -25,31 +30,31 @@ def test_calculate_A_tilde_inv_and_IFAinv(self):
     """
     epsilon = np.array([0.8, 0.4, 0.8])
     F = np.array([[0, 0.5, 0.5], [0.5, 0, 0.5], [0.5, 0.5, 0]])
-    expected_A_tilde_inv = np.array([
+    expected_a_tilde_inv = np.array([
         [0.83982684, 0.04761905, 0.11255411],
         [0.28571429, 0.42857143, 0.28571429],
         [0.11255411, 0.04761905, 0.83982684],
     ])
 
-    expected_IFAinv = np.array([
+    expected_ifainv = np.array([
         [0.64069264, -0.19047619, -0.45021645],
         [-0.19047619, 0.38095238, -0.19047619],
         [-0.45021645, -0.19047619, 0.64069264],
     ])
 
-    A_tilde_inv = utils.calculate_A_tilde_inv(epsilon, F)
-    IFAinv = utils.calculate_IFAinv(F, A_tilde_inv)
-    with self.subTest("A_tilde_inv shape"):
-      self.assertEqual(A_tilde_inv.shape, F.shape)
-    with self.subTest("IFAinv shape"):
-      self.assertEqual(IFAinv.shape, F.shape)
+    a_tilde_inv = utils.calculate_a_tilde_inv(epsilon, F)
+    ifainv = utils.calculate_ifainv(F, a_tilde_inv)
```

Comment:
Follow-up to previous comments about variable naming - our main goal is to be consistent. Here we see `a_tilde_inv` and `ifainv` , however we might consider using a variable name of `ifa_inv` or something, just to be consistent. This would apply to lots of the variables and parameter names.

But again this is not a major concern at this time, just a "nice to have" change.

Reply:
updated to ifa_inv